dataland-frontend/src/components/resources/frameworkDataSearch/TwoLayerDataTable.vue

```diff
@@ -51,26 +51,24 @@
             <span
               :data-test="`${reportingPeriodWithDataId.reportingPeriod}_${slotProps.data.categoryKey}_${slotProps.data.kpiKey}`"
             >
-              <template v-if="Array.isArray(slotProps.data.content[reportingPeriodWithDataId.dataId])">
-                <a
-                  v-if="
-                    slotProps.data.content[reportingPeriodWithDataId.dataId].length > 1 ||
-                    slotProps.data.content[reportingPeriodWithDataId.dataId].some((el) => typeof el === 'object')
-                  "
-                  @click="
-                    openModalAndDisplayValuesInSubTable(
-                      slotProps.data.content[reportingPeriodWithDataId.dataId],
-                      slotProps.data.kpiLabel,
-                      slotProps.data.kpiKey,
-                      slotProps.data.kpiFormFieldComponent,
-                    )
-                  "
-                  class="link"
-                  >Show "{{ slotProps.data.kpiLabel }}"
-                  <em class="material-icons" aria-hidden="true" title=""> dataset </em>
-                </a>
+              <template
+                v-if="
+                  slotProps.data.content[reportingPeriodWithDataId.dataId] &&
+                  isModal(slotProps.data.kpiFormFieldComponent)
```

Comment:
The new conditional does not do exactly the same as the old one. E.g. if the form field component was a multi select form field and only one entry was selected, the original version went into the else case and immediately displayed the single selected value. This new version would open a modal window in that case, right?

Reply:
Yes. It will open modal with one element in it.

Reply:
But who decided this change? Is this aligned with the team?

Reply:
Now everything that is an array is displayed in the modal